subsys/net/lib/lwm2m/lwm2m_rw_oma_tlv.c

```diff
@@ -971,7 +973,6 @@ int do_write_op_tlv(struct lwm2m_message *msg)
 			return ret;
 		}
 
-		return 0;
```

Comment:
Just trying to understand if it's ok to remove this return here. What happens in case of single-resource block writes? After processing the message content, we'll proceed to the loop below, and what then? `oma_tlv_get()` is expected to fail?

Reply:
exactly, in that case it will fail and break the loop.